test/command_line/test_generate_mask.py

```diff
@@ -153,3 +153,48 @@ def test_generate_mask_function_with_untrusted_rectangle(input_experiment_list,
         tmpdir.join(f).strpath for f in ("pixels4.mask", "pixels4_1.mask")
     ]
     assert imageset.external_lookup.mask.filename in associated_masks
+
+
+def test_generate_mask_trusted_range(dials_data, tmpdir):
+    # https://github.com/dials/dials/issues/978
+    image_files = dials_data("x4wide").listdir("*.cbf", sort=True)
+
+    # Import as usual
+    procrunner.run(
+        ["dials.import", "trusted_range=-1,100", "output.experiments=no-overloads.expt"]
+        + [f.strpath for f in image_files],
+        working_directory=tmpdir,
+    )
+    procrunner.run(
```

Comment:
I think it was discussed recently that it might be better to avoid the additional `procrunner.run()` level of indirection and instead call e.g. `dials.command_line.generate_mask.run()` directly. See e.g. https://github.com/dials/dials/commit/456755e334800c5bc2b5301c5300081c883f113c#diff-a57154b54de916bc084f9ff480d38c61R58